config.go

```diff
@@ -49,6 +48,7 @@ var (
 	btcwalletHomeDir   = btcutil.AppDataDir("btcwallet", false)
 	btcdHomedirCAFile  = filepath.Join(btcdHomeDir, "rpc.cert")
 	defaultConfigFile  = filepath.Join(btcwalletHomeDir, defaultConfigFilename)
+	defaultPubPassphrase = "public"
```

Comment:
This should probably be exported from another package.

Also, this file is not `go fmt`-formatted.